src/models/MilestoneModel.js

```diff
@@ -0,0 +1,289 @@
+import React from 'react';
```

Comment:
why the inconsistent naming of models? None of the existing models have the `Model` suffix

Reply:
Unfortunately, but we should for clearer declaration (.e.g `Milestone` the component vs `Milestone` the model)
Switched it back for now, don't want to spend time on renaming